src/main/kotlin/org/ionproject/integration/job/ISELTimetable.kt

```diff
@@ -0,0 +1,32 @@
+package org.ionproject.integration.job
+
+import org.ionproject.integration.step.tasklet.iseltimetable.implementations.DownloadAndCompareTasklet
+import org.springframework.batch.core.configuration.annotation.JobBuilderFactory
+import org.springframework.batch.core.configuration.annotation.StepBuilderFactory
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+
+@Configuration
+class ISELTimetable {
+
+    @Autowired
```

Comment:
Can't we use ctor injection here, instead of property injection?

Reply:
Updated it.